routers/books.py

```
@@ -0,0 +1,35 @@
+from schemas import schemas_books
+
+from typing import Optional
+
+from sqlalchemy.orm import Session
+
+from crud import crud_books
+
+from fastapi import Depends, HTTPException, APIRouter
+
+from session_db import db_session
+
+
+router = APIRouter()
+
+
+@router.get("/books/", response_model=list[schemas_books.GetBooks])
+def get_books_all(skip: int = 0, limit: int = 10, author_id: Optional[int] = None, db: Session = Depends(db_session)):
+    return crud_books.get_books(db, skip=skip, limit=limit, author_id = author_id)
+
+
+@router.get("/books/{book_id}", response_model=schemas_books.GetBooks)
+def get_books_id(book_id: int, db: Session = Depends(db_session)):
+    return crud_books.get_books_id(db, book_id)
```

This function doesn't handle the case where a book with the given `book_id` is not found. The `crud_books.get_books_id` function will return `None`, and FastAPI will raise a 500 error when trying to validate it against the `GetBooks` response model. 

You should add a check for the result and raise an `HTTPException` with a 404 status code if the book is not found.